### PR TITLE
[x86/Linux] PAL test in CI build

### PR DESCRIPTION
**Pull request by hseok-oh**
Add PAL test in CI x86/Linux build
Change docker image for build: including rootfs
New docker image for test

related issue: #9903

### CODE REVIEW

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu x86 Release Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test Ubuntu x86 Release Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu x86 Release Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu x86 Release Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu x86 Release Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu x86 Release Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test Ubuntu x86 Release Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test Ubuntu x86 Debug Build please
@dotnet-bot test Ubuntu x86 Checked Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu x86 Release Build please
@dotnet-bot test Ubuntu x86 Debug Build please
@dotnet-bot test Ubuntu x86 Checked Build please

**Comment by hseok-oh** [comment]
Ubuntu x86 Release/Checked failure: PAL test failure 

@gkhanna79 PTAL

**Comment by hseok-oh** [comment]
@dotnet-bot test this please
@dotnet-bot test Ubuntu x86 Release please
@dotnet-bot test Ubuntu x86 Checked please
@dotnet-bot test Ubuntu x86 Debug please
@dotnet-bot test ci please

**Comment by hseok-oh** [comment]
@dotnet-bot test Ubuntu x86 Release Build please
@dotnet-bot test Ubuntu x86 Checked Build please
@dotnet-bot test Ubuntu x86 Debug Build please

**Comment by gkhanna79** [comment]
@kouvel Can you please help take a look at the PAL test failure?

CC @janvorli

**Comment by janvorli** [comment]
@gkhanna79, @kouvel I will look into the issue unless @kouvel wants to become more familiar with the low level EH stuff and assembler unwinding annotations. 
@hseok-oh did this test ever pass on x86 Linux before? I think I have seen it always failing.

**Comment by janvorli** [comment]
I've forgotten to mention - it is the exception_handling/pal_sxs/test1/paltest_pal_sxs_test1 test failing with `terminate called after throwing an instance of 'PAL_SEHException'`, which means the exception was not caught by the native code. We have had this problem in this test on some platforms in the past for various reasons.

**Comment by hseok-oh** [comment]
For release build, I got same result as CI did - timeout. But for checked build, I passed pal test in my environment. Anyway, IMHO, It's better to enable PAL test for only debug build now, because implementation for x86 linux (checked/release build) is not enough yet.

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please
@dotnet-bot test Ubuntu x86 Release Build please
@dotnet-bot test Ubuntu x86 Debug Build please
@dotnet-bot test Ubuntu x86 Checked Build please

**Comment by janvorli** [comment]
@hseok-oh - now that I have fixed the release / checked build issue, do you want to re-enable the PAL tests for those build flavors or should I just merge this change as is?

**Comment by hseok-oh** [comment]
@janvorli I'll re-enable PAL test. Please wait merging.

**Comment by hseok-oh** [comment]
@dotnet-bot test Ubuntu x86 Release
@dotnet-bot test Ubuntu x86 Debug
@dotnet-bot test Ubuntu x86 Checked

**Comment by hseok-oh** [comment]
@gkhanna79 @janvorli x86/Linux PAL test succeeded. Please merge this.

File: netci.groovy [hunk 1]
```diff
@@ -1536,6 +1536,10 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                     if (architecture == 'x86' && os == 'Ubuntu') {
                         // build only, not test yet
                         buildCommands += "./tests/scripts/x86_ci_script.sh --buildConfig=${lowerConfiguration}"
+                        // And pal tests
+                        if (lowerConfiguration == "debug") {
```
Comment (janvorli) [inline netci.groovy]: Aren't we still adding the test results for debug only?

Comment (hseok-oh) [inline netci.groovy]: I've forgot to fix it. Thank you.

**Comment by hseok-oh** [comment]
@dotnet-bot test ci please

File: netci.groovy (hunk 1 above)
Comment (hseok-oh) [inline netci.groovy]: Fixed it. PTAL.

**Comment by janvorli** [comment]
@dotnet-bot test Ubuntu arm Cross Release Build please

**Pull request merged**